docs/source/content/supervised/knn.rst

```diff
@@ -62,9 +65,9 @@ Here is an example of how the K-D tree looks like.
 
    Ref: https://slideplayer.com/slide/3273367/
 
-How a K-D tree works is that a node in the tree represents and holds data from a n-dimensional
+How an K-D tree works is that a node in the tree represents and holds data from a n-dimensional
```

Comment:
This was the wrong a to switch, I meant the one at the end of the sentence.  The line should read: How a K-D tree works is that a node in the tree represents and holds data from an n-dimensional